testing/tests/DevExpress.viz.charts/chart.integration.tests.js

```diff
@@ -207,6 +207,112 @@ QUnit.test("Problem with two axis and range", function(assert) {
     assert.deepEqual(chart._valueAxes[0].getTicksValues().majorTicksValues, [0, 0.5, 1, 1.5, 2, 2.5, 3, 3.5, 4, 4.5, 5, 5.5, 6, 6.5, 7, 7.5, 8], "main value axis tick values");
 });
 
+QUnit.test("Expand/cut chart by panes (chart size not set)", function(assert) {
+    this.$container.height("");
+    var chartDrawn = sinon.spy();
+
+    var chart = this.createChart({
+        size: {
+            width: 840
+        },
+        dataSource: [{
+            arg: "1111111111111111",
+            val1: -10,
+            val2: 20
+        }, {
+            arg: "2222222222222222",
+            val1: 5,
+            val2: 3
+        }, {
+            arg: "5555555555555555",
+            val1: 7,
+            val2: 25
+        }, {
+            arg: "8888888888888888",
+            val1: 3,
+            val2: 5
+        }, {
+            arg: "11111111111111111111",
+            val1: 20,
+            val2: -10
+        }],
+        panes: [{ name: "p1", height: 200 }, { name: "p2", height: 250 }],
+        series: [{ valueField: "val1", pane: "p1" }, { valueField: "val2", pane: "p2" }],
+        argumentAxis: {
+            title: "Argument title",
+            label: {
+                overlappingBehavior: "rotate"
+            }
+        },
+        valueAxis: [{ pane: "p1" }, { pane: "p2" }],
+        title: "Simple chart",
+        legend: {
+            verticalAlignment: "bottom",
+            horizontalAlignment: "center"
+        },
+        onDrawn: chartDrawn
+    });
+
+    assert.roughEqual(chart.getSize().height, 630, 10, "Expand default height");
+
+    chart.option("panes[1].height", 200);
+    assert.roughEqual(chart.getSize().height, 580, 10, "Cut chart by pane height");
+
+    chart.option("size.width", 300);
+    assert.roughEqual(chart.getSize().height, 700, 10, "Expand height by argument axis content");
```

Comment:
Such an assertion seems not stable and depends on browser fonts rendering. I suggest moving this test in chartAxisDrawing.test.js where the environment allows setting axis margins to within px